docs/pim/create_product.md

```diff
@@ -0,0 +1,56 @@
+---
+description: Create new products or modify existing ones.
+---
+
+# Create products
+
+[Products](products.md#products) are a specific kind of [Content items](../content_management/content_items.md#content-items) that you use 
+to present your offer in the website, including product specification, and pricing.
+
+Individual products are instances of [product types](create_product_types.md#create-product-types).
+
+You can only create or modify products when your [user role](../permission_management/work_with_permissions.md) has the `Product/Edit` permission.
+
+To create a product, depending on the configuration, you [may need to provide](products.md#product-completeness) certain pieces of information, 
+such as a name, SKU, description and images, attributes, assets, availability, prices, and translations.
+
+You can create product by using manual or bulk method.
+Bulk method can be used only at the developer level. See [Products](https://doc.ibexa.co/en/latest/pim/product_api/#products) for a technical guide on how to do this.
+
+## Manually create product
+
+You can create product in several ways:
+
+1. Click **Create** in the upper-right corner of the [Products](products.md) screen. 
```

Comment:
I'd change this into a bulleted list - these not steps that need to be done in order, these are options to choose from.

Reply:
Ok, updated